ansible/debian-9-erlang-18.3.yml

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing,
+#   software distributed under the License is distributed on an
+#   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#   KIND, either express or implied.  See the License for the
+#   specific language governing permissions and limitations
+#   under the License.
+
+- hosts: couchdb-ci-worker
+  vars:
+    erlang_solutions_pkg_ver: "1:18.3-1"
```

Comment:
Why not do it for the for the version of erlang in stable. which is 19.2 
instead of 18.3 

https://tracker.debian.org/pkg/erlang

Reply:
As far as I understand there are two different builds. One with 18.3 (LTS?!) and one with the distribution default erlang (ansible/debian-9-erlang-default.yml) , 19.2 in that case.

Reply:
As far as I understand, the number 18.3 refers to the version of erlang in that particular distro. which in case of stretch is 19.2 . I may be wrong. but that is what I infer from the situation.

Reply:
We currently build against 18.3 in our CI pipeline and for released packages. When we move up to 19, we'll move to 19 across all releases, not just this one.

Reply:
understood. Thanks for the explanation.